writeengine/bulk/we_tableinfo.cpp

```diff
@@ -2232,78 +2234,31 @@ int TableInfo::validateColumnHWMs(
     int refCol = 0;
     int colIdx = 0;
 
-    //if (byte1First >= 0)
-    //  std::cout << "dbg: cross compare1 " << stage << " col-" << byte1First <<
-    //  "; wid-" << ( (jobTable != 0) ? jobTable->colList[byte1First].width :
-    //                                  fColumns[byte1First].column.width ) <<
-    //  "; hwm-" << segFileInfo[byte1First].fLocalHwm << std::endl;
-
-    //if (byte2First >= 0)
-    //  std::cout << "dbg: cross compare2 " << stage << " col-" << byte2First <<
-    //  "; wid-" << ( (jobTable != 0) ? jobTable->colList[byte2First].width :
-    //                                  fColumns[byte2First].column.width ) <<
-    //  "; hwm-" << segFileInfo[byte2First].fLocalHwm << std::endl;
-
-    //if (byte4First >= 0)
-    //  std::cout << "dbg: cross compare4 " << stage << " col-" << byte4First <<
-    //  "; wid-" << ( (jobTable != 0) ? jobTable->colList[byte4First].width :
-    //                                  fColumns[byte4First].column.width ) <<
-    //  "; hwm-" << segFileInfo[byte4First].fLocalHwm << std::endl;
-
-    //if (byte8First >= 0)
-    //  std::cout << "dbg: cross compare8 " << stage << " col-" << byte8First <<
-    //  "; wid-" << ( (jobTable != 0) ? jobTable->colList[byte8First].width :
-    //                                  fColumns[byte8First].column.width ) <<
-    //  "; hwm-" << segFileInfo[byte8First].fLocalHwm << std::endl;
-
     // Validate/compare HWMs given a 1-byte column as a starting point
     if (byte1First >= 0)
     {
         refCol = byte1First;
 
-        if (byte2First >= 0)
+        if ((rc = compareHWMs(byte1First, byte2First, 1, 2,
+             segFileInfo, colIdx) > 0))
```

Comment:
Looks like using `!= NO_ERROR` instead of `> 0` would be more understandable

Reply:
Good point indeed.